spec/models/piece_spec.rb

```diff
@@ -94,4 +92,43 @@
       end
     end
   end
+
+  describe '#move_to!' do
+    let(:piece) { build(:piece) }
```

Comment:
you could remove some duplication here by changing this to

```
let(:black_piece) { build(:piece, status: 'black') }
let(:white_piece) { build(:piece, status: 'black') }
```

Then use them below :)